TestAssets/TestProjects/TestProjectToProjectDependencies/src/L21/project.json

```diff
@@ -1,10 +1,10 @@
 {
   "version": "1.0.0-*",
   "dependencies": {
-    "NETStandard.Library": "1.5.0-rc2-24018"
+    "Microsoft.NETCore.App": "1.0.0-rc2-*"
   },
   "frameworks": {
-    "netstandardapp1.5": {
+    "netcoreapp1.0": {
```

Comment:
There are significant changes needed to the incremental tests in order to do this.

I agree that is technically correct, but given the intention of these tests is testing incremental build I don't believe it will have any impact.